packages/@webex/plugin-meetings/src/common/errors/webex-errors.ts

```diff
@@ -107,6 +107,27 @@ class UserInLobbyError extends WebexMeetingsError {
 export {UserInLobbyError};
 WebExMeetingsErrors[UserInLobbyError.CODE] = UserInLobbyError;
 
+/**
+ * @class SpaceIDDeprecatedError
+ * @classdesc Raised whenever the user passes Space ID as destination for create meeting.
+ * @extends WebexMeetingsError
+ * @property {number} code - 30105
+ * @property {string} message - Using the space ID as a destination is no longer supported. Please refer to the [migration guide](https://github.com/webex/webex-js-sdk/wiki/Migration-guide-for-USM-meeting) to migrate to use the meeting ID or SIP address.'
+ */
+class SpaceIDDeprecatedError extends WebexMeetingsError {
+  static CODE = 30105;
```

Comment:
Error code 30105 added as Space ID deprecated as of beta, Updated file packages/@webex/plugin-meetings/src/common/errors/webex-errors.ts